script.js

```diff
@@ -0,0 +1,360 @@
+    const splitString = convertedToString.split('');
+    const reverseString = splitString.reverse();
+    return reverseString.map(Number);
+}
+
+console.log(getReversedArrayOfDigits(35231));
+
+// 13. Write a function findNeedle() that takes an array full of junk but containing one "needle".
+
+function findNeedle(haystack) {
+    for (let i = 0; i < haystack.length; i++) {
+        if (haystack[i] === 'needle') {
+            return 'found the needle at position ' + i;
+        }
+    }
+}
+
+console.log(findNeedle(["hay", "junk", "hay", "hay", "moreJunk", "needle", "randomJunk"]));
+
+//  14. Given an array of integers, return a new array with each value doubled.
+// For example:
+// [1, 2, 3] --> [2, 4, 6]
+
+function getDoubledArray(arrayOfNumbers){
+    let doubledArray = [];
+    for (let i = 0; i < arrayOfNumbers.length; i++) {
+        const duplicatedValue = arrayOfNumbers[i] + arrayOfNumbers[i];
+        doubledArray.push(duplicatedValue);
+    }
+    return doubledArray;
+}
+
+console.log(getDoubledArray([1, 2, 3]));
+
+//  15. Given a set of numbers, return the additive inverse of each. Each positive becomes negatives, and the negatives become positives.
+
+function getInvertedArray(array) {
+    let invertedArray = [];
+    for (let i = 0; i < array.length; i++) {
+        const invertedNumber = array[i] * -1;
+        invertedArray.push(invertedNumber);
+    }
+    return invertedArray;
+}
+
+console.log(getInvertedArray([1, -2, 3, -4, 5]));
+
+// 16. Write a function that takes an array of numbers and returns the sum of the numbers.
+// The numbers can be negative or non-integer.
+// If the array does not contain any numbers then you should return 0.
+
+function getSumOfNumbers(numbers) {
+    let sumOfNumbers = 0;
+    if (numbers.length === 0) {
+        return 0;
+    }
+    for (let i = 0; i < numbers.length; i++) {
+        sumOfNumbers = sumOfNumbers + numbers[i];
+    }
+    return sumOfNumbers;
+}
+
+console.log(getSumOfNumbers([]));
+
+// 17. I'm new to coding and now I want to get the sum of two arrays... Actually the sum of all their elements. I'll appreciate for your help.
+// P.S. Each array includes only integer numbers. Output is a number too.
+
+// create the getArraySum function that sums the elements of a single array and use it in the arrayPlusArray function.
+
+function getArraySum(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+        sum += array[i];
+    }
+    return sum;
+}
+
+function getTheSumOfTwoArrays(firstArray, secondArray) {
+    const firstArraySum = getArraySum(firstArray);
+    const secondArraySum = getArraySum(secondArray);
+    return firstArraySum + secondArraySum;
+}
+
+console.log(getTheSumOfTwoArrays([1, 2, 3], [4, 5, 6]));
+
+// 18. Given a non-negative integer, 3 for example, return a string with a murmur: "1 sheep...2 sheep...3 sheep...".
+// Input will always be valid, i.e. no negative integers.
+
+function getSheepCount (number){
```

Comment:
There is an unnecessary space in this line.
Also, the variable name `number` is too vague.